src/pages/recipes/index.css

```diff
@@ -1,11 +1,18 @@
+* {
+  margin: 0;
+}
+
 .recipe {
-	background-color: white;
+  background-color: white;
 	padding: 20px;
 	margin: 10px;
+  border-radius: .5em;
 }
 
 .recipe-title {
-	font-size: 1.5em;
+  display: flex;
```

Comment:
is the flex property needed here?

Reply:
No, `display: flex` should only be in the parent class. Thanks for catching that, I'll update this branch